modules/ROOT/pages/salesforce/salesforce-connector-studio.adoc

```diff
@@ -0,0 +1,165 @@
+= Salesforce - Studio
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+== Install Salesforce  
+
+. In Anypoint Studio, click the Exchange icon in the Studio task bar.
+. Click Login in Anypoint Exchange and supply your Anypoint Platform username and password.
+. Search for the Salesforce connector and click Install.
+. Follow the prompts to install the connector.
+
+== Configure this Connector
```

Comment:
Don't use a pronoun as there's no antecedent in a title. Configure Salesforce Connecter or similar?